Makefile

```diff
@@ -261,6 +262,8 @@ test-cpp: build-test-cpp
 
 test-java: build-test-java
 	echo "Running tests against $(LIB_PROFILER)"
+	# generate copies of libasyncProfiler for testing multiple profilers
+	cp -f build/$(LIB_PROFILER) build/test/lib/libasyncProfiler.$(SOEXT)
```

Comment:
Some other test already copies `libasyncProfiler` manually. This should be unified.